synapse/lib/stormtypes.py

```diff
@@ -2467,6 +2467,9 @@ def getObjLocals(self):
         }
 
     async def _libBytesUpload(self, genr):
+
```

Comment:
please add entries for these in the perms declarations

Reply:
(The main item of interest there being that default=True)